script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt

```diff
@@ -170,7 +171,8 @@ Ultimately, the Luminary had only taken damage unintentionally.
 [sha:ccdfb369106aa103520e380c56f8cf3b45f1c4e9]{
 -- Page 294, Offset 41115.
 -- 直前に手酷い目に遭わせられたカルヴァリアの痛みの記憶に、ロアが持つ『アルクェイド本人の力』が組み合わされたものだったからだ。
-The reason it was even possible was the memory of the intense pain she'd suffered at the hands of Calvaria earlier coupled with the original fraction of her power that Roa had carried.
+The reason it was even possible was the memory of the intense pain she'd suffered at the hands of Calvaria earlier,
+ coupled with the original fraction of her power that Roa had carried.
```

Comment:
This page still is a bit messy. Honestly very difficult.

alt
>Ultimately, the Luminary had taken damage unconsciously———

>Something only possible due to the combination of her painful memories of Calvaria from just before together with the original fraction of her power carried by Roa.

alt
>It was the combination of her memory of the intense pain caused by Calvaria earlier,
>and the original fraction of her power held by Roa———

>That caused the Luminary to unconsciously take damage.

Hmmph, it all feels weird. God what a terrible line

Reply:
Maybe add more structure so that it can then be split?
> Ultimately, the Luminary had taken damage unconsciously———
> Something only possible as a result of a convergence of factors:
>  The still-fresh memories of the pain inflicted on her by Calvaria,
>  and the original fraction of her power carried by Roa.

Reply:
sure